jest.config.mjs

```diff
@@ -0,0 +1,23 @@
+import nextJest from 'next/jest.js'
+ 
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+})
+ 
+// Add any custom config to be passed to Jest
+/** @type {import('jest').Config} */
+const config = {
+  // Automatically clear mock calls, instances and results before every test
+  clearMocks: true,
+  // Indicates whether the coverage information should be collected while
+  // executing the test
+  collectCoverage: true,
+  // Add more setup options before each test is run
+  // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
```

Comment:
Do create this file (and try making it a `.ts` file) and it can be similar to this one: https://github.com/NYPL/research-catalog/blob/main/jest.setup.js

With this file, you can remove line 6 from `__tests__/index.test.tsx` and place it in the setup file. It's not a problem, but it becomes tedious when you have to import that line for every test file.